training/training/core/dataset.py

```diff
@@ -98,3 +101,75 @@ def getCategoryList(self) -> list[str]:
         if self._category_list is None:
             raise Exception("Category list not available")
         return self._category_list
+
+
+class DefaultImageDatasets(Enum):
+    MNIST = "MNIST"
+    FASHION_MNIST = "FashionMNIST"
+    KMNIST = "KMNIST"
```

Comment:
is this real? @codingwithsurya

Reply:
yea i imported torchvision datasets

Reply:
i see. so KMNIST is an actual dataset. interesting. What about the CIFAR dataset? can u add that

Reply:
sure